frontend/src/features/contacts/contact/edit/Person/UpdatePersonForm.tsx

```diff
@@ -146,6 +163,34 @@ const UpdatePersonComponent: React.FC<FormikProps<IEditablePersonForm>> = ({
     }
   };
 
+  // update mailing address sub-form when "useOrganizationAddress" checkbox is toggled
+  useEffect(() => {
+    // toggle is on - set mailing address values to match organization address
+    if (useOrganizationAddress === true && organizationId) {
+      getOrganization(organizationId)
+        .then(({ data }) => getApiMailingAddress(data))
```

Comment:
Is there a benefit to having multiple thens here?

Reply:
no benefit, personal preference to single lines but can be re-written as:

```tsx
  getOrganization(...)
    .then(({data)) => {
      const mailing = getApiMailingAddress(...)
      setFieldValue(..., mailing)
    })
   .catch(...)

```